src/index.html

```diff
@@ -5,9 +5,317 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Не розумію, навіщо тут два однакових рядки
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
```